File: Java/Easy/Accepted/length-of-last-word_7.java
class Solution {
    public int lengthOfLastWord(String s) {
        
        int length = s.length();
        int lastWordLength = 0;
        boolean lastWordStarted = false;
        for (int i = length - 1; i >= 0; i--) {
            if (s.charAt(i) == ' ') {
                if (lastWordStarted) {
                    return lastWordLength;
                }
            } else {
                lastWordLength++;
                lastWordStarted = true;
            }
        }
        return lastWordLength;
    }
}

File: C++/Easy/Accepted/length-of-last-word_7.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        
        int len = s.length();
        int i = len - 1;
        int cnt = 0;
        while (i >= 0 && s[i] == ' ') i--;
        while (i >= 0 && s[i] != ' ') {
            i--;
            cnt++;
        }
        return cnt;
    }
};